lib/ZumoOta/src/BootloaderCom.cpp

```diff
@@ -336,10 +581,25 @@ class FwProgCmdProvider : public CmdProvider {
  * @brief Name of the firmware file to be used for programming. 
  * This variable holds the name of the firmware file ("/zumo_firmware.bin")
  * that will be used for programming the device.
+ * In the finalversion this should be provided by the Webserver
+ */
+ std::string fileName = "/firmware.bin";
+ File m_firmwareFile; /**< Handle for the firmware file.*/
+ const char* m_fileName = fileName.c_str(); /**< File name for the firmware file.*/
+
+/** 
+ * The size of each data chunk used for processing, set to 128 bytes.
  */
-std::string fileName = "/firmware.bin";
+static const size_t chunkSize = 128U;
+
+/** 
+ * Buffer to hold data read from a file, with a size of 128 bytes.
+ */
+uint8_t mybuffer[chunkSize];

```

Comment:
myBuffer is a pointless name.  Why is this global ? It should be static or part of a class (static).